src/reference/asciidoc/kafka.adoc

```diff
@@ -295,6 +315,27 @@ IMPORTANT: When `producerPerThread` is `true`, user code **must** call `closeThr
 This will physically close the producer and remove it from the `ThreadLocal`.
 Calling `reset()` or `destroy()` will not clean up these producers.
 
+When creating a `DefaultKafkaProducerFactory`, key and/or value `Serializer` classes can be picked up from configuration by calling the constructor that only takes in a Map of properties (see example in <<kafka-template>>),
+or `Serializer` instances may be passed to the `DefaultKafkaProducerFactory` constructor (in which case all Producers share the same instances).
+Alternatively you can provide Suppliers of Serializers that will be used to obtain separate `Serializer` instances for each `Producer`:
```

Comment:
Don't "`Serializer`s" and "`Serializer`s" work here?

Reply:
Ah good, I wasn't sure - will apply the highlighting